gnocchi/tests/base.py

```diff
@@ -268,9 +269,15 @@ class TestCase(BaseTestCase):
     @classmethod
     def setUpClass(self):
         super(TestCase, self).setUpClass()
-        self.conf = service.prepare_service([],
-                                            default_config_files=[])
-        if not os.getenv("GNOCCHI_TEST_DEBUG"):
+
+        debug = "GNOCCHI_TEST_DEBUG" in os.environ
+
+        self.conf = service.prepare_service(
+            [],
+            default_config_files=[],
+            logging_level=logging.DEBUG if debug else logging.NOTSET)
```

Comment:
It's pretty well explained here https://docs.python.org/2/library/logging.html

What happens if that we set the log level to NOTSET, the logging level of the handler above (the root one in this case) is used.

There's actually not really any interest of setting NOTSET, since the output is removed anyway. I'll update the patch to simplify.